helm/alfresco-process-infrastructure/values.yaml

```diff
@@ -481,6 +481,9 @@ alfresco-deployment-service:
     image:
       # alfresco-deployment-service.applications.image.tag -- default tag for all images used in application
       tag: develop
+      pullPolicy: IfNotPresent
+      # alfresco-deployment-service.applications.image.pullPolicy -- pull policy for alfresco-deployment-service image
```

Comment:
for consistency this comment goes above the line and
"pull policy for alfresco-deployment-service image"
change to
"default pull policy for all images used in application"

sounds good?

Reply:
yes its updated now. 
thanks